api/python/flaschen.py

```diff
@@ -14,10 +14,32 @@
 
 import socket
 
+# Netpbm raw RGB header.
+_HEADER_P6 = b"""\
+P6
+%(width)d %(height)d
+255
+"""
+
+# Flaschen Taschen footer holding the offset.
+_FOOTER_FT = b"""\
+%(x)d
+%(y)d
+%(z)d
+"""
```

Comment:
Now we don't need this format string anymore.

Reply:
I've removed this, and the unused header.